lib/taurus/core/taurusmanager.py

```diff
@@ -121,7 +121,7 @@ def addJob(self, job, callback=None, *args, **kw):
         """ Deprecated. Wrapper of enqueueJob. See enqueueJob documentation.
         """
         self.deprecated(dep='addJob', alt='enqueueJob', rel='4.3.2')
-        self.enqueueJob(job, callback=callback, jobargs=args, jobkwargs=kw)
+        self.enqueueJob(job, callback, args, kw)
```

Comment:
I do not agree with this change. It is better to be explicit and use kwargs, so the previous version is preferable.